assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/stable_diffusion/requirements.txt

```diff
@@ -0,0 +1,5 @@
+mlflow==2.6.0
```

Comment:
can we use conda.yaml over requirements as it allows pinning of things like which python version to use and few more packages which is not available via pip (not relevant in this case)? eg: 
```
channels:
- anaconda
- pytorch
- conda-forge
dependencies:
- pip<=22.2.2
- python=3.8.13
- pip:
   - torch==1.13.0
   - transformers==4.29.1
   - diffusers==0.20.0
   - accelerate==0.22.0
name: mlflow-env
```